staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -37,5 +37,12 @@ ${CODEGEN_PKG}/generate-internal-groups.sh "deepcopy,defaulter,conversion" \
   --output-base "$(dirname ${BASH_SOURCE})/../../.." \
   --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt
 
+openapi-gen --logtostderr \
```

Comment:
this should go to https://github.com/kubernetes/code-generator/blob/master/generate-internal-groups.sh. Would that make sense?

Reply:
Yes, it makes perfect sense.  That is what your PR #55472 set out to do.  I died due to lack of activity.  Should it be re-opened?